stream/offset/offset_latest_seqno_init.go

```diff
@@ -0,0 +1,27 @@
+package offset
+
+import (
+	"github.com/Trendyol/go-dcp/config"
+)
+
+type OffsetLatestSeqNoInit struct {
+	config *config.Dcp
+}
+
+func NewOffsetLatestSeqNoInit(
+	config *config.Dcp,
+) *OffsetLatestSeqNoInit {
+	return &OffsetLatestSeqNoInit{
+		config,
+	}
+}
+
+func (l *OffsetLatestSeqNoInit) InitializeLatestSeqNo(LatestSeqNo uint64) uint64 {
+	var latestSeqNo uint64
+	if l.config.IsDcpModeFinite() {
+		latestSeqNo = LatestSeqNo
+	} else {
+		latestSeqNo = 0xffffffffffffffff
```

Comment:
We can exctract the value of `0xffffffffffffffff` to appropriate variable. It will more readable for future readers.